packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -322,19 +424,27 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
       return;
     }
     if (event is PointerMoveEvent || event is PointerPanZoomUpdateEvent) {
-      final Offset delta = (event is PointerMoveEvent) ? event.delta : (event as PointerPanZoomUpdateEvent).panDelta;
-      final Offset localDelta = (event is PointerMoveEvent) ? event.localDelta : (event as PointerPanZoomUpdateEvent).localPanDelta;
-      final Offset position = (event is PointerMoveEvent) ? event.position : (event.position + (event as PointerPanZoomUpdateEvent).pan);
-      final Offset localPosition = (event is PointerMoveEvent) ? event.localPosition : (event.localPosition + (event as PointerPanZoomUpdateEvent).localPan);
       if (_state == _DragState.accepted) {
-        _checkUpdate(
-          sourceTimeStamp: event.timeStamp,
-          delta: _getDeltaForDetails(localDelta),
-          primaryDelta: _getPrimaryValueFromOffset(localDelta),
-          globalPosition: position,
-          localPosition: localPosition,
-        );
+        _pointerMoveAccept = true;
+        // When a same pointer is detected, update immediately and add this event.
+        final bool eventExists = _multiPointerMoveTrackers.any((PointerEvent element) => element.pointer == event.pointer);
+        if (eventExists) {
```

Comment:
Do we still need to check or track `eventExists`? We probably should just trust `endOfBatch`.

Reply:
> Do we still need to check or track `eventExists`? We probably should just trust `endOfBatch`. This will make the algorithm much simpler and easier to explain.

On Android, if p1 and p2 down and only p1 leaves, then another p3  down (here are 3 pointers totally) and moves, the p3' **endOfBatch always false**. Because there are 2 pointer on the screen. So this logic is still useful.

Reply:
Is it the same issue as https://github.com/flutter/flutter/pull/106552#discussion_r935535606?